src/types.ts

```diff
@@ -32,6 +31,14 @@ export type RequireOnlyOne<T, Keys extends keyof T = keyof T> = Pick<
     [K in Keys]-?: Required<Pick<T, K>> & Partial<Record<Exclude<Keys, K>, undefined>>;
   }[Keys];
 
+export type RequireOnlyTwo<T, Keys extends keyof T = keyof T> = Pick<
```

Comment:
Is it used somewhere?

Reply:
nah, we thought originally there'd be only 2 translation texts `{ en_text: 'Something', es_text: 'Translated' }` but there can be more than that. but after this type rule was figured out so we left it just in case it's needed in the future for anything as it was a bit of a thought process to get to

Reply:
Since it's not in master, might be good to remove.

Reply:
@ferhatelmas @vishalnarkhede removed